### DISCUSSION TITLE
Is implicit converstion from Maybe<T> to Nullable<T> possible?

### DISCUSSION THREAD

**Discussion by SuperJMN** [vkhorikov/CSharpFunctionalExtensions#564] [Q&A]
Imagine I have:

```charp
int? number = 123;
var maybeNumber = Maybe.From(1);
```

Why can't I just do this?

```charp
number = maybeNumber;
```

This would be quite handy for some common scenarios.

**Comment by vkhorikov** [comment]
Yeah, that's a good suggestion. I'm not 100% the compiler would allow this, but it's worth a try.
